canonical-web-and-design/vanilla-framework
# Fix docs heading levels

## Done

- Promoted every heading on every doc page by one level, so that there's a `h1` and appropriate heirarchy on each page.
- Updated script that creates sublists in the side nav to look for `h2` elements, rather than `h3`

## QA

- Open [demo](https://vanilla-framework-3939.demos.haus/docs)
- Go to any (or all, if you're feeling brave) docs page
- Use the headingsMap extension ([Firefox](https://addons.mozilla.org/en-GB/firefox/addon/headingsmap/)/[Chrome](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en)), and see that it doesn't flag issues with the heading heirarchy. If there are, it will highlight the problems:

![Screenshot from 2021-08-19 11-40-45](https://user-images.githubusercontent.com/2376968/130055437-65da8cff-2459-4bda-bc0e-7319eddd9e00.png)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
